modules/icon/css/stories.tsx

```diff
@@ -2,45 +2,111 @@ import React, {Component} from 'react';
-    return (
-      <div>
+    return <div className={containerStyle}>{this.props.children}</div>;
+  }
+}
+
+storiesOf('CSS/Icon', module)
+  .addDecorator(withReadme(README))
+  .add('Accent Icon', () => (
+    <div className="story">
+      <IconDemo>
+        <i className="wdc-icon" data-icon="shield" data-category="accent" />
+        <i
+          className="wdc-icon"
+          data-icon="shield"
+          data-category="accent"
+          data-color="pomegranate500"
+        />
+        <span
+          style={{
+            backgroundColor: colors.blueberry500,
+            display: 'inline-block',
+            verticalAlign: 'top',
+          }}
+        >
+          <i
+            className="wdc-icon"
+            data-icon="shield"
+            data-category="accent"
+            data-color="frenchVanilla100"
+            data-transparent="true"
+          />
+        </span>
+        <br />
+        <i className="wdc-icon" data-icon="shield" data-category="accent" data-size="80" />
+      </IconDemo>
+    </div>
+  ))
+  .add('Applet Icon', () => (
+    <div className="story">
+      <IconDemo>
+        <i className="wdc-icon" data-icon="benefits" data-category="applet" />
+        <i
+          className="wdc-icon"
+          data-icon="benefits"
+          data-category="applet"
+          data-color="pomegranate"
+        />
+        <br />
+        <i className="wdc-icon" data-icon="benefits" data-category="applet" data-size="60" />
+      </IconDemo>
+    </div>
+  ))
+  .add('System Icon', () => (
+    <div className="story">
+      <IconDemo>
+        <i className="wdc-icon" data-icon="activityStream" data-category="system" />
         <i
           className="wdc-icon"
           data-icon="activityStream"
           data-category="system"
-          data-fill-color="blueberry"
-          data-accent-color="blueberry"
-          data-background-color="transparent"
+          data-fill-color="blueberry500"
+          data-accent-color="blueberry500"
         />
         <i
           className="wdc-icon"
           data-icon="activityStream"
           data-category="system"
-          data-fill-color="cinnamon300"
-          data-accent-color="cinnamon300"
-          data-background-color="transparent"
+          data-fill-color="blueberry500"
+          data-accent-color="frenchVanilla100"
+          data-background-color="blueberry500"
         />
-        <i className="wdc-icon" data-icon="activityStream" data-category="system" />
-        <i className="wdc-icon" data-icon="apple" data-category="accent" data-color="chiliMango" />
-        <i className="wdc-icon" data-icon="rocket" data-category="applet" data-hue="cinnamon" />
-      </div>
-    );
-  }
-}
-
-storiesOf('CSS/Icon', module)
-  .addDecorator(withReadme(README))
-  .add('All', () => (
-    <div className="story">
-      <IconDemo />
+        <br />
+        <i className="wdc-icon" data-icon="activityStream" data-category="system" data-size="48" />
```

Comment:
Can we add an example with hover colors?

Reply:
Sounds good. I have a handful of stories I was using to test the hover colors anyway -- I'll use those.

Reply:
Quick question: should I add stories with non-default hover colors to the React System Icon as well? Or does that belong in a separate PR?

Reply:
You could probably do it here - that's fine 👍